### ISSUE TITLE
Access Control bug?

### ISSUE THREAD

**Issue by Gawcyk** [jflamy/owlcms4#494]
When set Password
![ustawianie](https://user-images.githubusercontent.com/24601725/172476428-b5d77939-8bad-4149-b0ed-e322b112463b.png)
and clicking Update then 
![poaktualizacji](https://user-images.githubusercontent.com/24601725/172476510-173add09-e31d-45c0-9d46-51899250a805.png)

This is correct??

**Comment by Gawcyk** [comment]
Ok this is good. But when i'm set better password and i trying login to the system i See incorect password

**Comment by jflamy-dev** [comment]
I have just tried with the current version (4.32) and it worked both on a local setup and on Heroku.   Set password, Update.  Close browser, go back to site, login ok.  Also tried just typing a new password again.

If you are locked out, on Heroku you can set the application OWLCMS_PIN to (empty) -- no value.  
On the local application, edit the owlcms.l4j file and add -DOWLCMS_PIN=
to the file (define the personal identification to be empty).  This takes precedence over the stored password

**Comment by jflamy-dev** [comment]
Correction: for the local application, it's -Dpin=

**Comment by jflamy-dev** [comment]
Typing too quickly not being clear.  Let me try again.
-Dpin=
that defines the Java system property, and you can add that line to the owlcms.l4j.ini file in the desktop application install directory (.ini may or may not be visible depending on your Windows settings if running on Windows)

On Heroku, each application has configuration variables under the Settings page
OWLCMS_PIN
should be set to empty.

On Linux, you can either set the environment variable OWLCMS_PIN, or add -Dpin to the java invocation.
